packages/core/package.json

```diff
@@ -118,7 +119,7 @@
   },
   "codegenConfig": {
     "name": "RNSentrySpec",
-    "type": "modules",
+    "type": "all",
```

Comment:
Q: why this change?

Reply:
There are 3 types `all`, `modules` and `components`.

`modules` -> Native Module
`components` -> Native UI Components
`all` -> Combination of above.

Reply:
Thank you for the explanation :)